vsintegration/tests/FSharp.Editor.Tests/Refactors/RemoveExplicitReturnType.fs

```diff
@@ -0,0 +1,68 @@
+﻿module FSharp.Editor.Tests.Refactors.RemoveExplicitReturnType
+
+open Microsoft.VisualStudio.FSharp.Editor
+open Xunit
+open System
+open System.Collections.Immutable
+open System.Text.RegularExpressions
+
+open Microsoft.CodeAnalysis
+open Microsoft.CodeAnalysis.CodeFixes
+open Microsoft.CodeAnalysis.Text
+open Microsoft.VisualStudio.FSharp.Editor
+open Microsoft.VisualStudio.FSharp.Editor.CancellableTasks
+
+open FSharp.Compiler.Diagnostics
+open FSharp.Editor.Tests.Helpers
+
+open Microsoft.CodeAnalysis.CodeRefactorings
+open NUnit.Framework
+open Microsoft.CodeAnalysis.CodeActions
+open System.Collections.Generic
+open Microsoft.VisualStudio.LanguageServices
+open FSharp.Editor.Tests.Refactors.RefactorTestFramework
+open Microsoft.Build.Utilities
+open System.Threading
+open FSharp.Test.ReflectionHelper
+open Microsoft.Build.Utilities
+open FSharp.Test.ProjectGeneration.ProjectOperations
+open FSharp.Compiler.Symbols
+
+[<Fact>]
+let ``Refactor changes something`` () =
+    task {
+
+        let code =
+            $"""
+            let sum a b :int= a + b
```

Comment:
Good test - would be also useful to play with the spacing here, to assert that things work for ` : int`, `: int`, ` :    int` and so on.

Reply:
@psfinaki What would be helpful: Do you have a list of all the different ways, one can define a method? I would add tests for all the variations here : https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/functions/ and here : https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/members/methods , but are there other styles that could make problems?

Reply:
Hmm I'd say for this particular question the docs should be / are supposed to be exhaustive. 

That said, as Brian mentions in a [sibling PR](https://github.com/dotnet/fsharp/pull/16079), such things have "combinatorially-explosive nature" :) So the strategy here is:
1) Cover the obvious/common/whatever-comes-to-mind cases in the beginning
2) Add problematic and edge cases as followups when they show up
3) Be conservative and defensive in VS logic - it's better to not suggest a fix/refactoring at all than suggest something that in some cases breaks customer code